src/ui/sass/_base.scss

```diff
@@ -1,15 +1,14 @@
-*:focus
```

Comment:
Are we sure we want to use component? It seems different from our standard scoped classes. Also seems like this would be a very popular class that SDK users would use

Reply:
I picked it because that's the class we were already including by default on all containers. If we want to use a different class name, we can, but it can't replace the `component` default (removing that would be breaking).

Reply:
for sure

i agree its easier to use what's already there but i think there's some value to choosing a class that won't interfere with client code (e.g. `yxt-Component` or `yxt-Answers-component`). it feels easier for us to say our contract is "all `yxt` prefaced classes are the classes we are adding styling to", instead of "all yxt + 'component'"`. i just have the /feeling/ that `component` would be used a lot, esp. in react environments.

i also could not find why we keep the component class (no css referencing the class, not in any js files i saw). do we use that? not important, just something i noticed.

either way, i think the accreting method of adding an answers specific css class makes it easier to talk about our effects. what do you think?

Reply:
I think you're right. We've said before our convention is to prefix classnames with `yxt`. That helps provide collisions and makes it clear to customers where the HTML is coming from. I don't think we've encountered issues so far with our default of `component`, but once we enter React world, we probably will. 

The one thing I don't like is that now people will see divs with the classes `component` and `yxt-Answers-component`. If I were a customer, I'd ask why we need to proprietary class names. Adding the class name `component` was likely a mistake. Unfortunately, it's not one we can remove until a new major version is released. I say we live with this annoyance for now and eye removing the `component` class name in the future. I don't want us to continue adding two classes to every container. 

We don't use the class name of `component` anywhere in the SDK, we provided that as a hook for client styling. Unfortunately, we have no way of know if customers are relying on this class name (they probably are).